src/components/MovieItem/index.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+import './style.css';
+
+class MovieItem extends React.Component {
+    constructor() {
+        super()
```

Comment:
Not necessary to keep constructor if you only call super().

Reply:
fixed